src/lib/NetworkHelper.ts

```diff
@@ -0,0 +1,6 @@
+export function isDevNetwork() {
+  return [
+    '1001',
+    '1002',
+  ].includes(process.env.NETWORK_ID);
```

Comment:
This might not be a good way to check this --- I think `NETWORK_ID` is only used in one place (`test/helpers/Perpetual.ts`) and it might not be set as expected.

Can we use `perpetual.networkId`?

Btw I think we can eventually get rid of the NETWORK_ID env variable, it's one of the changes I made in this abandoned PR: https://github.com/dydxprotocol/perpetual/pull/62/files